### PR TITLE
update readme with usage guidance.

### LINKED ISSUE #1: Add "how to run on MiniKube"
The README.md file contains only "How to Build". We also need "How to Run".

### PR DESCRIPTION
**Pull request by putcn**
fix: #1 
Please do NOT merge until #7 is merged.
will add training job creating guidance.

### CODE REVIEW

**Review by helinwang — DISMISSED** [review]
Thank you! LGTM++!

File: README.md [hunk 1]
```diff
@@ -32,3 +32,147 @@ of EDL on this cluster is
...
+
+``` bash
+cd example
+docker build -t yourRepoName/my_edl_training_job .
+```
+
+then push it to docker hub to be accessible by Kubernetes:
+
+``` bash
+docker push yourRepoName/my_edl_training_job
```
Comment (typhoonzero) [inline README.md:151]: Maybe add `git clone` command before head so that readers are not confused by `yourRepoName`

**Review by typhoonzero — APPROVED** [review]
LGTM, only one minor comment.

File: README.md (hunk 1 above)
Comment (putcn) [reply README.md:151]: agreed, will update with "Resource Adjustments by EDL" part and FAQ in next PR

**Pull request merged**